Bottom: 642a0086ec29df8ce45f9f8d981bf303d97f1472
Top:    7e158c6c97b7855af2f8ec91f0c17139358d2361
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-20 03:17:50 +0000

Refresh of other-drawing-api-changes

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 44b7916..8c544d3 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -2180,10 +2180,9 @@ hid_gtk_init ()
   ghid_hid.notify_save_pcb          = ghid_notify_save_pcb;
   ghid_hid.notify_filename_changed  = ghid_notify_filename_changed;
 
-#if 0
-  ghid_hid.graphics                 = &ghid_graphics;
+//  ghid_hid.graphics                 = &ghid_graphics;
 
-  common_draw_helpers_init (&ghid_graphics);
+//  common_draw_helpers_init (&ghid_graphics);
 
   ghid_graphics.make_gc             = ghid_make_gc;
   ghid_graphics.destroy_gc          = ghid_destroy_gc;
@@ -2197,7 +2196,6 @@ hid_gtk_init ()
   ghid_graphics.fill_circle         = ghid_fill_circle;
   ghid_graphics.fill_polygon        = ghid_fill_polygon;
   ghid_graphics.fill_rect           = ghid_fill_rect;
-#endif
 
   hid_register_hid (&ghid_hid);
 #include "gtk_lists.h"
